.github/workflows/deploy_tre.yml

```diff
@@ -5,8 +5,8 @@ name: Deploy Azure TRE
 
 on:  # yamllint disable-line rule:truthy
   schedule:
-    # 1am each night https://crontab.guru/#0_1_*_*_*
-    - cron: "0 1 * * *"
+    # 1am each night https://crontab.guru/#0_0_*_*_*
+    - cron: "0 0 * * *"
```

Comment:
Did you disable nightly build on purpose?

Reply:
No. What makes you think that?

Reply:
Sorry, my mistake.

Reply:
just need to update the 1am to midnight in the comment.